Marlin/Configuration_adv.h

```diff
@@ -1284,11 +1284,12 @@
 // Additional options for DGUS / DWIN displays
 //
 #if HAS_DGUS_LCD
-  #define DGUS_SERIAL_PORT 2
+  #define DGUS_SERIAL_PORT 3
   #define DGUS_BAUDRATE 115200
 
   #define DGUS_RX_BUFFER_SIZE 128
   #define DGUS_TX_BUFFER_SIZE 48
+  #define SERIAL_TX_BUFFER_SIZE 256
```

Comment:
This define is not used anywhere in Marlin. There is a `TX_BUFFER_SIZE`, however. It's defined later in this same file.

Reply:
This is used in ststm32 framework.

Reply:
But defining a same-named thing here has no influence on the STM32 framework. Its .CPP files cannot see it.

Reply:
My mistake. Delete it.